stdlib/account.go

```diff
@@ -1669,35 +1666,35 @@ func changeAccountContracts(
 
 	// Validate the contract update
 
-	inter := invocation.Interpreter
+	inter := invocation.InvocationContext
 
 	if isUpdate {
 		oldCode, err := handler.GetAccountContractCode(location)
 		handleContractUpdateError(err, newCode)
 
-		memoryGauge := invocation.Interpreter.SharedState.Config.MemoryGauge
-		legacyUpgradeEnabled := invocation.Interpreter.SharedState.Config.LegacyContractUpgradeEnabled
+		memoryGauge := invocation.InvocationContext
+		//legacyUpgradeEnabled := invocation.InvocationContext.SharedState.Config.LegacyContractUpgradeEnabled
 
-		var oldProgram *ast.Program
+		//var oldProgram *ast.Program
```

Comment:
Wasn't sure whether we still support this. So commented out for now (one less thing to refactor).
If we do need this, I'll bring it back.

Reply:
AFAICS and AFAIK this is no longer used/needed (e.g. see https://github.com/search?q=org%3Aonflow%20LegacyContractUpgradeEnabled&type=code). Let's just remove it

Reply:
Removed https://github.com/onflow/cadence/pull/3826/commits/0e5c9afbd83f822f203cfe4e6ee2c17c3fd98709